TODO: make it configurable via dist tag in configure.ac

--- NetworkManager-0.8.9997/src/settings/plugins/ifcfg-rh/common.h~	2011-05-31 10:37:37.000000000 +0300
+++ NetworkManager-0.8.9997/src/settings/plugins/ifcfg-rh/common.h	2011-08-19 11:07:06.325518503 +0300
@@ -36,7 +36,9 @@
 #define AUGNEW_TAG ".augnew"
 #define AUGTMP_TAG ".augtmp"
 
-#define IFCFG_DIR SYSCONFDIR"/sysconfig/network-scripts"
+#ifndef IFCFG_DIR
+#define IFCFG_DIR SYSCONFDIR"/sysconfig/interfaces"
+#endif
 
 #define IFCFG_PLUGIN_NAME "ifcfg-rh"
 #define IFCFG_PLUGIN_INFO "(c) 2007 - 2010 Red Hat, Inc.  To report bugs please use the NetworkManager mailing list."
